dashboard/app/models/user.rb

```diff
@@ -1185,6 +1185,11 @@ def self.find_for_authentication(tainted_conditions)
     login = conditions.delete(:login)
     if login.present?
       return nil if login.size > max_credential_size || login.utf8mb4?
+      Cdo::Metrics.put(
+        'User', 'LoginByUsername', 1, {
+          Environment: CDO.rack_env
+        }
+      )
```

Comment:
It would be less distracting if we could put it on one line:
```suggestion
      Cdo::Metrics.put('User', 'LoginByUsername', 1, Environment: CDO.rack_env)
```